components/nifosg/nifloader.cpp

```diff
@@ -1030,6 +1030,78 @@ namespace NifOsg
             }
         }
 
+        class PreciseLeafDepthCullCallback : public osg::DrawableCullCallback{
```

Comment:
`osg::DrawableCullCallback` is `typedef`ed to `osg::Drawable::CullCallback`, and `osg::Drawable::CullCallback` existed in older OSG versions, so switching to that should help with the compilation issues with 3.4.1.

Reply:
I see you fixed this while I was typing that out.